src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -7573,6 +7574,10 @@ static void writeStaffDetails(XmlWriter& xml, const Part* part)
             }
             xml.startElement("staff-details", attributes);
 
+            if (st->links() && st->links()->contains(part->staff(i - 1))) {
```

Comment:
One more thing: wouldn't this crash when `i == 0`? Maybe a check like this is necessary:
```suggestion
            if (i > 0 && st->links() && st->links()->contains(part->staff(i - 1))) {
```

Reply:
No it should't, but having this additional check wouldn't hurt.